.github/workflows/core.yml

```diff
@@ -41,17 +41,33 @@ jobs:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Build the tagged Docker image
-        run: docker-compose -f compose/core-${{ matrix.r_version }}.yml build
+        run: |
+          docker-compose -f compose/core-${{ matrix.r_version }}.yml build
+          docker-compose -f compose/shiny-${{ matrix.r_version }}.yml build
+          docker-compose -f compose/geospatial-${{ matrix.r_version }}.yml build
+          docker-compose -f compose/binder-${{ matrix.r_version }}.yml build
       - name: Push the tagged Docker image
-        run: docker-compose -f compose/core-${{ matrix.r_version }}.yml push
+        run: |
+          docker-compose -f compose/core-${{ matrix.r_version }}.yml push
+          docker-compose -f compose/shiny-${{ matrix.r_version }}.yml push
+          docker-compose -f compose/geospatial-${{ matrix.r_version }}.yml push
+          docker-compose -f compose/binder-${{ matrix.r_version }}.yml push
       - name: Push latest tagged Docker image
         if: matrix.r_latest == true
         run: |
           docker image tag rocker/r-ver:${{ matrix.r_version }} rocker/r-ver:latest
           docker image tag rocker/rstudio:${{ matrix.r_version }} rocker/rstudio:latest
           docker image tag rocker/tidyverse:${{ matrix.r_version }} rocker/tidyverse:latest
           docker image tag rocker/verse:${{ matrix.r_version }} rocker/verse:latest
+          docker image tag rocker/verse:${{ matrix.r_version }} rocker/shiny:latest
```

Comment:
looks like these tags were not updated

Reply:
@cboettig I apologize for not noticing this mistake. Since the build won't run if you just fix this part, could you merge #176?